moccasin/config.py

```diff
@@ -107,8 +109,11 @@ def _set_boa_env_and_db(self) -> _AnyEnv:
 
         # 3. Finally, "true" networks
         elif self.is_zksync:
-            env = ZksyncEnv(EthereumRPC(self.url), nickname=self.name)
-            boa.set_env(env)
+            if self.explorer_type != "zksyncexplorer":
```

Comment:
the string should be in a constant. Btw, why is it lower case here?
If imported, we could also use `ZksyncExplorer.__name__`

Reply:
They are able to set the `explorer_type` in the `moccasin.toml`. It's hard to know what type of explorer it is based on the URL alone.